themes/stanford_basic/stanford_basic.theme

```diff
@@ -117,6 +117,58 @@ function stanford_basic_page_attachments_alter(array &$attachments) {
   }
   // Check if dropdown menus are activated
   $attachments['#attached']['drupalSettings']['stanford_basic']['nav_dropdown_enabled'] = (bool) theme_get_setting('nav_dropdown_enabled', 'stanford_basic');
+
+  // Only add Google Analytics for anonymous users or users without editing
+  // access while on Acquia environment. Being able to view the toolbar is a
+  // good indicator that they can edit some part of the site.
```

Comment:
@pookmish thanks for these changes. 
@buttonwillowsix, does this logic match how we are using our GA tracking code? 

Would this opt-out most users on the intranet sites?

Reply:
@buttonwillowsix ping.